src/Generator.js

```diff
@@ -218,10 +257,10 @@ const trimCucumberKeywords = (name, ...i18nkeys) => {
   return name.slice(charsToTrim).trim();
 };
 
-const getFeatureButtons = (features) => {
+const getFeatureButtons = (featureFileTree) => {
   const featureButtons = [];
-
-  features.forEach((feature) => {
+  featureFileTree.children.filter((child) => child.type === 'file').forEach((child) => {
+    const { feature } = child;
```

Comment:
```suggestion
  featureFileTree.children
    .filter((child) => child.type === 'file')
    .forEach((child) => {
      const { feature } = child;
```

and then indent the rest below